gui/src-tauri/src/main.rs

```diff
@@ -281,7 +328,15 @@ fn main() -> Result<()> {
 			// WindowEvent::Resized(_) => std::thread::sleep(std::time::Duration::from_nanos(1)),
 			_ => (),
 		})
-		.build(tauri_context);
+		.build(context)
+}
+
+fn tauri_build_result(
+	build_result: Result<tauri::App, tauri::Error>,
+	exit_flag: Arc<AtomicBool>,
+	backend: Arc<Mutex<Option<CommandChild>>>,
+) {
+	let backend_termination = backend.clone();
```

Comment:
there is no need to clone this as u just use it on one part of the code

Reply:
Yeah... Found that part of the original code kinda weird.